components/profile/pages/voting-power/oracles/delegate/controller.tsx

```diff
@@ -61,7 +63,7 @@ export default function OraclesDelegate({
       clearTimeout(debounce.current);
 
       debounce.current = setTimeout(() => {
-        const isValid = Service.active.isAddress(params.target.value);
+        const isValid = isAddress(params.target.value);
```

Comment:
redundancy, we never use `isValid` again, we can simply make the `if (isAddress(params.target.value))`